models/netsuite/netsuite__income_statement.sql

```diff
@@ -67,19 +67,19 @@ income_statement as (
         subsidiaries.name as subsidiary_name
 
         --The below script allows for accounts table pass through columns.
-        {{ fivetran_utils.persist_pass_through_columns('accounts_pass_through_columns', identifier='accounts') }},
+        {{ netsuite.persist_pass_through_columns(var('accounts_pass_through_columns', none), identifier='accounts') }},
```

Comment:
Is there a reason we use None here and not `[]` like we usually do?

Reply:
Especially since that's how we have it defined in the [dbt_project.yml](https://github.com/fivetran/dbt_netsuite/blob/19261a9274ca8a6b5e1fabd7e5083e417422b857/dbt_project.yml#L60)

Reply:
Oh, [I was following what was defined in our persist passthrough macro code](https://github.com/fivetran/dbt_fivetran_utils/blob/releases/v0.4.latest/macros/persist_pass_through_columns.sql#L3) and assumed the none condition was what we wanted. I'll update this.